internal/server/server.go

```diff
@@ -36,12 +39,16 @@ func (s *Server) Init() *Server {
 		s.router = http.NewServeMux()
 	}
 
-	s.router.Handle("/metrics", metrics.Handler())
-	s.router.HandleFunc("GET /health", s.health)
+	apiPrefix := s.apiPrefix
+
+	s.log.Debugf("Api prefix set to '%s'", apiPrefix)
```

Comment:
If we add a debug log, I'd make it something that contains all the information we can provide. Something like _"Listening on <url>:<port>/<prefix>"_. What do you think? Alternatively, I'd drop the log completely.

Reply:
Ok, providing this info in one place is not perfect, because address is sitting in params and and prefix resides in config. So i will drop it for now.